src/components/VUETable.tsx

```diff
@@ -1,44 +1,73 @@
-                <tr>
-                    <td>{info.hugoGeneSymbol}</td>
-                    <td>{info.genomicLocation}</td>
-                    <td>{info.defaultEffect}</td>
-                    <td>{info.comment}</td>
-                    <td>{info.context}{info.referenceText && (<>{' '}<a href={`https://pubmed.ncbi.nlm.nih.gov/${info.pubmedIds[0]}/`} rel="noreferrer" target="_blank">({info.referenceText})</a></>)}</td>
-                    <td>{info.revisedProteinEffects && (<a href={`https://deploy-preview-139--genome-nexus-frontend.netlify.app/variant/${info.revisedProteinEffects[0].variant}`} rel="noreferrer" target="_blank">Genome Nexus <i className="fa fa-external-link" /></a>)}</td>
-                </tr>
-            );
-        });
+export const VUETable: React.FC = () => {
+    const [vueData, setVueData] = useState<VUE[]>([]);
+
+    useEffect(() => {
+        const fetchData = async () => {
+          const data = await fetchVueData();
+          setVueData(data);
+        };
+
+        fetchData();
+    }, []);
+
+    const DisplayData = vueData.map((info) => {
+        const references: { referenceText: string, pubmedId: number }[] = [];
+        info.revisedProteinEffects.map(v => ({ referenceText: v.referenceText, pubmedId: v.pubmedId })).filter(r => {
+            let i = references.findIndex(ref => (ref.referenceText === r.referenceText && ref.pubmedId === r.pubmedId))
+            if (i === -1) {
+                references.push(r)
+            }
+            return null;
+        })
+
+        const links = references.map<React.ReactNode>(reference => 
+            (<>
+                <a href={`https://pubmed.ncbi.nlm.nih.gov/${reference.pubmedId}/`} rel="noreferrer" target="_blank">
+                    ({reference.referenceText})
+                </a>
+            </>)
+        ).reduce((prev, curr) => [prev, ', ', curr])
 
         return (
-            <div>
-                <table className="table table-striped">
-                    <thead>
-                        <tr>
-                            <th>Gene</th>
-                            <th>Genomic Location</th>
-                            <th>Predicted Effect</th>
-                            <th>
-                                <img alt='reVUE logo' src={vueLogo} width={20} style={{marginBottom:2}} />{' '}
-                                Actual Effect
-                            </th>
-                            <th>Context & Reference</th>
-                            <th>Usage Example</th>
-                        </tr>
-                    </thead>
-                    <tbody>{DisplayData}</tbody>
-                </table>
-            </div>
+            <tr>
+                <td>{info.hugoGeneSymbol}</td>
+                <td>{info.genomicLocationDescription}{info.revisedProteinEffects.length > 1 && <Link style={{ marginLeft: '10px' }} to={`/vue/${info.hugoGeneSymbol}`} state={info}><button className='btn-sm'>View All</button></Link>}</td>
```

Comment:
Personally I think the button is too bold, maybe good to make it less noticeable, like the style we have for referenceLink 
<img width="153" alt="image" src="https://github.com/knowledgesystems/reVUE/assets/16869603/2c5ef9db-b1d8-46ea-9164-6428f928f7b0">